api/queue.py

```diff
@@ -0,0 +1,73 @@
+from django.db import connection
+
+from api.models import TaskDef, Task
+
+get_task_sql = """
```

Comment:
Is there something that raw SQL is giving you that isn't available via the django queryset api?